core/Command/User/LastSeen.php

```diff
@@ -31,55 +33,75 @@
 use Stecman\Component\Symfony\Console\BashCompletion\CompletionContext;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
-class LastSeen extends Base {
-	public function __construct(
-		protected IUserManager $userManager,
-	) {
-		parent::__construct();
-	}
+class LastSeen extends Base
+{
+    public function __construct(
+        protected IUserManager $userManager,
+    ) {
+        parent::__construct();
+    }
 
-	protected function configure() {
-		$this
-			->setName('user:lastseen')
-			->setDescription('shows when the user was logged in last time')
-			->addArgument(
-				'uid',
-				InputArgument::REQUIRED,
-				'the username'
-			);
-	}
+    protected function configure()
+    {
+        $this
+            ->setName('user:lastseen')
+            ->setDescription('shows when the user was logged in last time')
+            ->addArgument(
+                'uid',
+                InputArgument::OPTIONAL,
+                'the username'
+            )
+            ->addOption(
+                'all',
+                null,
+                InputOption::VALUE_NONE,
+                'shows a list of when all users were last logged in'
+            )
+        ;
+    }
 
-	protected function execute(InputInterface $input, OutputInterface $output): int {
-		$user = $this->userManager->get($input->getArgument('uid'));
-		if (is_null($user)) {
-			$output->writeln('<error>User does not exist</error>');
-			return 1;
-		}
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $singleUserId = $input->getArgument('uid');
+        if ($singleUserId) {
```

Comment:
```suggestion
        if ($singleUserId !== null) {
```
What if the user id is `'0'` ? :stuck_out_tongue: